subprojects/core/src/main/java/org/gradle/api/internal/tasks/DefaultTaskInputs.java

```diff
@@ -204,7 +206,7 @@ public void visitInputProperty(String propertyName, PropertyValue value, boolean
             }
 
             @Override
-            public void visitInputFileProperty(final String propertyName, boolean optional, boolean skipWhenEmpty, boolean incremental, @Nullable Class<? extends FileNormalizer> fileNormalizer, PropertyValue value, InputFilePropertyType filePropertyType) {
+            public void visitInputFileProperty(final String propertyName, boolean optional, boolean skipWhenEmpty, DirectorySensitivity directorySensitivity, boolean incremental, @Nullable Class<? extends FileNormalizer> fileNormalizer, PropertyValue value, InputFilePropertyType filePropertyType) {
```

Comment:
I think it would be nice to split these parameter declarations to separate lines now.

Reply:
Agreed.  I've cleaned this up in the various implementors.